package.json

```diff
@@ -56,6 +59,7 @@
     "jest-environment-jsdom": "^29.7.0",
     "npm-run-all": "^4.1.5",
     "rollup": "^4.24.0",
+    "semantic-release": "^24.1.2",
```

Comment:
Let's stick with npx. You can use a specific version via `npx semantic-release@24.1.2`.